lib/components/Editor/index.js

```diff
@@ -74,8 +74,16 @@ const Tiptap = (
   const addonOptions = addons.map((option) => option.toLowerCase());
   isUnsplashImageUploadActive && addonOptions.push("image-upload");
   const allOptions = defaultOptions.concat(addonOptions);
-  const formattedValue =
-    typeof value === "string" ? value.replaceAll(" ", "&nbsp;") : "";
+
+  // Replace all spaces within tags with "nbsp;". This prevents tiptap editor from removing trailing spaces while typing.
+  let formattedValue = "";
```

Comment:
@AbhayVAshokan Can we move this to a util function. `index` is already too crowded